src/Callout/Callout.mdx

```diff
@@ -0,0 +1,119 @@
+    augue sit amet justo.
+  </Callout>
+  <Callout type="danger" title="This is a title" marginBottom="20px">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse diam ipsum, cursus id placerat congue, ultrices
+    eget lectus. Duis posuere, lacus sed tristique commodo, sapien turpis mollis nunc, vestibulum consectetur lectus
+    augue sit amet justo.
+  </Callout>
+  <Callout type="warning" title="This is a title" marginBottom="20px">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse diam ipsum, cursus id placerat congue, ultrices
+    eget lectus. Duis posuere, lacus sed tristique commodo, sapien turpis mollis nunc, vestibulum consectetur lectus
+    augue sit amet justo.
+  </Callout>
+</Playground>
+
+## Tints
+
+You can add a tint using the `hasTint` prop.
+
+<Playground>
+  <Callout hasTint type="info" title="This is a title" marginBottom="20px">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse diam ipsum, cursus id placerat congue, ultrices
+    eget lectus. Duis posuere, lacus sed tristique commodo, sapien turpis mollis nunc, vestibulum consectetur lectus
+    augue sit amet justo.
+  </Callout>
+  <Callout hasTint type="success" title="This is a title" marginBottom="20px">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse diam ipsum, cursus id placerat congue, ultrices
+    eget lectus. Duis posuere, lacus sed tristique commodo, sapien turpis mollis nunc, vestibulum consectetur lectus
+    augue sit amet justo.
+  </Callout>
+  <Callout hasTint type="danger" title="This is a title" marginBottom="20px">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse diam ipsum, cursus id placerat congue, ultrices
+    eget lectus. Duis posuere, lacus sed tristique commodo, sapien turpis mollis nunc, vestibulum consectetur lectus
+    augue sit amet justo.
+  </Callout>
+  <Callout hasTint type="warning" title="This is a title" marginBottom="20px">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse diam ipsum, cursus id placerat congue, ultrices
+    eget lectus. Duis posuere, lacus sed tristique commodo, sapien turpis mollis nunc, vestibulum consectetur lectus
+    augue sit amet justo.
+  </Callout>
+</Playground>
+
+## Icon
+
+You can also specify your own [`<Icon>` component](/components/icon) using the `icon` prop.
+
+<Playground>
+  <Callout icon="flame" title="This is a title">
+    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse diam ipsum, cursus id placerat congue, ultrices
+    eget lectus. Duis posuere, lacus sed tristique commodo, sapien turpis mollis nunc, vestibulum consectetur lectus
+    augue sit amet justo.
+  </Callout>
+</Playground>
+
+## Footer
+
+You can render a footer for the callout using the `footer` prop.
+
+<Playground>
+  <Callout type="info" hasTint title="This is a title" footer={<ActionButtons justifyContent="flex-end" width="100%" />}>
```

Comment:
I was thinking maybe we should add `submitProps={{ palette: 'info' }}` to <ActionButtons> to keep consistent with the color.